src/plugins/ComputedPlugin.js

```diff
@@ -0,0 +1,78 @@
+import React from 'react';
+
+/**
+ * A plugin for a computed
+ */
+function ComputedPlugin() {
+  const name = 'computed';
+
+  /**
+   * Augment the base schema with the computed type
+   * @param {*} schema
+   */
+  const augmentSchema = ((schema) => {
+    const additions = {
+      inlines: {
+        computed: {
+          nodes: [{
+            match: { object: 'text' }
```

Comment:
This is fine for now... but I think in theory a computed value could be any block of markdown... @jeromesimeon do you confirm?

Reply:
That's correct, although we will need to review that aspect in more details when we revise the Cicero parser. This code means we are assuming computed variables are inlines, so I think this is fine for now.